docs-web/technical-specifications.md

```diff
@@ -0,0 +1,91 @@
+# NGINX Ingress Controller Technical Specifications
+
+## Supported Kubernetes Versions
+
+The NGINX Ingress Controller has been verified to run on the following Kubernetes versions:
+* Kubernetes 1.14-1.18
+
+## Supported Docker Images
+
+We provide the following Docker images, which include NGINX/NGINX Plus bundled with the Ingress Controller binary. 
```

Comment:
Not sure about the wording here, but I can't think of another way to say it at the moment.